packages/withRequired/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "@apollo/utils.withrequired",
+  "version": "1.0.0",
```

Start at 0 with a changeset `major`
```suggestion
  "version": "0.0.0",
```